root/frontend/src/components/PreviewCard/PreviewCard.tsx

```diff
@@ -1,38 +1,36 @@
 import React from 'react';
-import { useTheme } from 'react-jss';
 import { BsFillSuitDiamondFill } from 'react-icons/bs';
 import { HiDotsHorizontal } from 'react-icons/hi';
-import useStyles from './PreviewCard.style';
+import '../../styles/PreviewCard.scss';
 
 type PropTypes = {
   title: string;
   exercises: string[];
+  theme: string;
   //  placeholder until logos are figured out
   //  logo: string
 };
 
-function PreviewCard({ title, exercises }: PropTypes) {
-  const theme = useTheme();
-  const classes = useStyles({ theme });
+function PreviewCard({ title, exercises, theme }: PropTypes) {
   const listExercise = exercises.map((exercise) => (
-    <li className={classes.listExerciseItem}>
-      <div className={classes.exerciseItem}>
-        <BsFillSuitDiamondFill className={classes.diamond} />
-        <p className={classes.exercise}>{exercise}</p>
+    <li className="listExerciseItem">
+      <div className="exerciseItem">
+        <BsFillSuitDiamondFill style={{ color: theme }} />
```

Comment:
Looking at the icons, I think we may want to go with `BsDiamondFill` instead. Personally, I find this icon looks like it's in the wrong resolution.